apps/client/src/hoc/withAutofillData.js

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from "react";
+import { useHistory } from "react-router-dom";
+import { Redirect } from "react-router-dom";
+
+import withChecker from "./withChecker";
+import Context from "../context";
+import { routes, geturl, autofillRoutes } from "../routes";
+import { isEmptyObject } from "../utils";
+import { autofillResolvers, autofillMap } from "../autofill";
+
+const withAutofillData = (Component) =>
+  withChecker((props) => {
+    const history = useHistory();
+    const { autofillData } = useContext(Context);
```

Comment:
Not for now, but lets be consistent, with the destructs from useContext. 
I perfer this way above const context = useContext(Context)

Reply:
Please explain why in dev-meeting. I'm curious.